spec/core_functions/inspect.hrx

```diff
@@ -369,6 +369,26 @@ a {
   type: string;
 }
 
+<===>
+================================================================================
+<===> list/nested/options.yml
+---
+:todo:
+- libsass # sass/libsass#2654
+
+<===> list/nested/input.scss
+a {
+  $a: inspect([() (())]);
```

Comment:
Why is this `(())` here? That's no different than `()`, and if we're testing that those two are parsed identically, that should probably go in the specs for lists rather than the specs for `inspect()`.

Reply:
I'm not sure. This is for https://github.com/sass/libsass/issues/2654.

The important bit is:

```
@debug $example; //=> [() ()]
@debug inspect($example); //=> [(() ())]
```

So it does seem to be specific to inspect.

Reply:
I would just get rid of the extra parentheses around the second list, then:

```suggestion
  $a: inspect([() ()]);
```

Reply:
Thanks, done in https://github.com/sass/sass-spec/commit/18b91d4abdc9ad6f6a8b90f1945ed3e5b95c583c